File: Examples/test-suite/python/namespace_class_runme.py
from namespace_class import *

try:
  p = Private1()
  error = 1
except:
  error = 0

if (error):
  raise RuntimeError, "Private1 is private"

try:
  p = Private2()
  error = 1
except:
  error = 0

if (error):
  raise RuntimeError, "Private2 is private"

EulerT3D.toFrame(1,1,1)

b = BooT_i()
b = BooT_H()


f = FooT_i()
f.quack(1)

f = FooT_d()
f.moo(1)

f = FooT_H()
f.foo(Hi)

f_type = str(type(f))
if f_type.find("'namespace_class.FooT_H'") == -1:
  raise RuntimeError("Incorrect type: " + f_type)

File: Examples/test-suite/matlab/namespace_class_runme.m
try
  p = namespace_class.Private1();
  ok = 1;
catch
  ok = 0;
end

if (ok)
  error('Private1 is private')
end

try
  p = namespace_class.Private2();
  ok = 1;
catch
  ok = 0;
end

if (ok)
  error('Private2 is private')
end

namespace_class.EulerT3D.toFrame(1,1,1);

b = namespace_class.BooT_i();
b = namespace_class.BooT_H();


f = namespace_class.FooT_i();
f.quack(1);

f = namespace_class.FooT_d();
f.moo(1);

f = namespace_class.FooT_H();
f.foo(namespace_class.Hi);

